d3a_interface/settings_validators.py

```diff
@@ -37,54 +42,53 @@ def validate_global_settings(settings_dict):
-            ConstSettings.IAASettings.MARKET_TYPE_LIMIT.max):
-        raise D3ASettingsException(f"Invalid value ({settings_dict['spot_market_type']}) "
+                                   f"({settings['cloud_coverage']}).")
+    if ("spot_market_type" in settings
+            and not ConstSettings.IAASettings.MARKET_TYPE_LIMIT[0]
+            <= settings["spot_market_type"]
+            <= ConstSettings.IAASettings.MARKET_TYPE_LIMIT[1]):
+        raise D3ASettingsException(f"Invalid value ({settings['spot_market_type']}) "
                                    "for spot market type.")
-    if "sim_duration" in settings_dict and not slot_length <= settings_dict["sim_duration"]:
+    if ("bid_offer_match_algo" in settings
+            and not ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE_LIMIT[0]
+            <= settings["bid_offer_match_algo"]
+            <= ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE_LIMIT[1]):
+        raise D3ASettingsException(f"Invalid value ({settings['bid_offer_match_algo']}) "
+                                   "for bid offer match algo.")
+    if "sim_duration" in settings and not slot_length <= settings["sim_duration"]:
         raise D3ASettingsException("Invalid simulation duration "
                                    f"(lower than slot length of {slot_length.minutes} min")
-    if "market_count" in settings_dict and not 1 <= settings_dict["market_count"]:
+    if "market_count" in settings and settings["market_count"] < 1:
         raise D3ASettingsException("Market count must be greater than 0.")
-    if ("capacity_kW" in settings_dict and not
+    if ("grid_fee_type" in settings and
+            int(settings["grid_fee_type"]) not in ConstSettings.IAASettings.VALID_FEE_TYPES):
+        raise D3ASettingsException("Invalid value for grid_fee_type "
+                                   f"({settings['grid_fee_type']}).")
+    if ("capacity_kW" in settings and not
             ConstSettings.PVSettings.CAPACITY_KW_LIMIT.min
-            <= settings_dict["capacity_kW"]
+            <= settings["capacity_kW"]
             <= ConstSettings.PVSettings.CAPACITY_KW_LIMIT.max):
         raise D3ASettingsException("Invalid value for capacity_kW "
-                                   f"({settings_dict['capacity_kW']}).")
-    if ("grid_fee_type" in settings_dict and
-            int(settings_dict["grid_fee_type"]) not in ConstSettings.IAASettings.VALID_FEE_TYPES):
-        raise D3ASettingsException("Invalid value for grid_fee_type "
-                                   f"({settings_dict['grid_fee_type']}).")
-
-    if ("relative_std_from_forecast_percent" in settings_dict and not
-            PercentageRangeLimit.min <=
-            settings_dict["relative_std_from_forecast_percent"] <=
-            PercentageRangeLimit.max):
-        raise D3ASettingsException("Invalid value for relative_std_from_forecast_percent "
-                                   f"({settings_dict['relative_std_from_forecast_percent']}).")
```

Comment:
I think these were recent addition in context to settlement market. Why you want to remove it?

Reply:
It looks like these were messed up in the merge conflict, I'll fix them